index.html

```diff
@@ -0,0 +1,99 @@
+<head>
```

Comment:
Nice general structure overall, but is this a semantically correct HTML document? You are missing a doctype and actual HTML elements.